CoreDistributedCache/NHibernate.Caches.CoreDistributedCache.Tests/CoreDistributedCacheFixture.cs

```diff
@@ -35,6 +37,7 @@ public partial class CoreDistributedCacheFixture : CacheFixture
 	{
 		protected override bool SupportsSlidingExpiration => true;
 		protected override bool SupportsClear => false;
+		protected override bool SupportsDistinguishingKeysWithSameStringRepresentationAndHashcode => false;
```

Comment:
Supporting this for a distributed cache was a mistake. (Maybe even testing for this support is a mistake: I am not sure it has actually any valid use.)